homeassistant/components/tibber/sensor.py

```diff
@@ -383,12 +387,14 @@ def _set_state(self, state, timestamp):
         ]:
             self._attr_last_reset = timestamp.replace(
                 hour=0, minute=0, second=0, microsecond=0
-            )
+            ).astimezone(timezone.utc)
```

Comment:
Please use this util:
https://github.com/home-assistant/core/blob/ebf6e3d985a2cb75333554cb411b23001258be08/homeassistant/util/dt.py#L69